recipes/diligent-core/all/conanfile.py

```diff
@@ -48,29 +50,28 @@ def validate(self):
             check_min_cppstd(self, self._minimum_cpp_standard)
         min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
         if not min_version:
-            self.output.warn("{} recipe lacks information about the {} compiler support.".format(
+            self.output.warning("{} recipe lacks information about the {} compiler support.".format(
                 self.name, self.settings.compiler))
         else:
             if Version(self.settings.compiler.version) < min_version:
                 raise ConanInvalidConfiguration("{} requires C++{} support. The current compiler {} {} does not support it.".format(
                     self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
-        if self.settings.compiler == "Visual Studio" and "MT" in self.settings.compiler.runtime:
+        if visual.is_msvc_static_runtime(self):
```

Comment:
```suggestion
        if visual.is_msvc(self) and visual.is_msvc_static_runtime(self):
```

Reply:
Is this really necessary? `is_msvc_static_runtime` already checks for `is_msvc`.

Reply:
Is this really necessary? `is_msvc_static_runtime` already checks for `is_msvc`.

Reply:
Oh great catch, I didn't realize! TIL :)